### ISSUE TITLE
Docker Storage Lab

### ISSUE THREAD

**Issue by ManoMarks** [docker/labs#118]
Create a new storage lab, vender agnostic

**Comment by eliassal** [comment]
Mano, in the settings, I have ticked the E: drive as a shared volume.
Does this mean that this volume is available for all linux containers? I ran 
`docker run -i -t centos /bin/bash
`once in bash, I run
`ls   `   I get

> [root@2b4a2318883e /]# ls
> anaconda-post.log  bin  dev  etc  home  lib  lib64  lost+found  media  mnt  opt  proc  root  run  sbin  srv  sys  tmp  usr  var
> 

I don't see the E volume

So do I need to issue somthing per container?
Thanks

**Comment by eliassal** [comment]
So do I need to issue somthing per container before running the container?

**Comment by ManoMarks** [comment]
@eliassal This isn't the place for that question :-) This is an issue to track creating a labs tutorial. If you're having issues with Docker for Windows, you can post them here: https://github.com/docker/for-win or in the Docker Forums https://forums.docker.com/

**Comment by eliassal** [comment]
Thanks Mano, I thought I was in the docker for win forum, sorry for that, while waiting for the response I found a good thread on the volume subject
https://forums.docker.com/t/volume-mounts-in-windows-does-not-work/10693/23
it is huge but very useful. The only thing I was not able to figure out is how to add volumes in a Dockerfile. I visited 
https://docs.docker.com/engine/tutorials/dockervolumes/#mount-a-host-directory-as-a-data-volume
but no example of doing this. Anyway, thanks for your response and I will post the question at docker/for-win
